.ci_support/osx_64_.yaml

```diff
@@ -0,0 +1,41 @@
+MACOSX_DEPLOYMENT_TARGET:
+- '10.9'
```

Comment:
I have no idea if this is relevant for the build process, but 10.9 is ancient. The oldest macOS version that still gets security updates is 10.13.

Reply:
I found this in the documentation: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cmake#requiring-newer-macos-sdks

Reply:
I see, thanks for clarifying. Then, we should stick with 10.9. The code shouldn't have anything that requires a more recent macOS version.

Reply:
I can give it a try just to see if it does anything

Reply:
This doesn't seem to be the problem